code/modules/mob/living/carbon/xenomorph/castes/defender/abilities_defender.dm

```diff
@@ -147,6 +147,44 @@
 		X.use_plasma(plasma_cost)
 	add_cooldown()
 
+// ***************************************
+// *********** Forward Charge
+// ***************************************
+/datum/action/xeno_action/activable/forward_charge
+	name = "Forward Charge"
+	action_icon_state = "charge"
+	mechanics_text = "Charge up to 4 tiles and knockdown any targets in your way."
+	ability_name = "charge"
+	cooldown_timer = 15 SECONDS
+	plasma_cost = 80
+
+/datum/action/xeno_action/activable/forward_charge/can_use_ability(atom/A, silent = FALSE, override_flags)
+	. = ..()
+	if(!.)
+		return FALSE
+	if(!A)
+		CRASH("forward_charge/can_use_ability invoked without an atom/A")

```

Comment:
shouldn't crash here just return false.

Reply:
also crashing here without at least a . = FALSE will maybe result in can_use_ability returning TRUE